homework-submission/my-project/src/AddForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+
+class AddTodo extends React.Component {
+    state = {
+        description: '',
+        deadline: '',
+        done: false,
+
+    }
+    newTodo = (e) => {
+        this.setState({
+            [e.target.id]: e.target.value
+        })
+        console.log(this.state)
+    }
+    newDate = (date) => {
+        this.setState({
+            deadline: date.format("YYYY-MM-DD"),
+        })
+    }
+    submit = (e) => {
+        e.preventDefault();
+        this.props.addTodo(this.state)
+        this.setState({
+            description: '',
+            deadline: '',
+            done: false,
+
+        })
+    }
+    render() {
+        return (
+            <div className="newitem">
+                <form onSubmit={this.submit}>
+                    <span className="add">Add todo</span>
+                    <div >
+                        <span>Enter description : </span>
+                        <input type='text'
+                            className="newtodo"
+                            id='description'
+                            onChange={this.newTodo} />
```

Comment:
Make sure to handle your value changes here in order to reset them and having a better control over them correctly.